src/util.js

```diff
@@ -0,0 +1,31 @@
+import F from 'futil'
+import _ from 'lodash/fp'
+
+export let tokenizePath = path =>
```

Comment:
You already have a futil tree configured, so you can just use the futil methods (e.g. Tree.lookup, etc)

Reply:
It turns out I still need this fn, because `remove` takes a string like `remove('path.to.["dotted.field"]')` and passing an array to remove all the time is cumbersome